api/extension-guides/language-model-tutorial.md

```diff
@@ -9,13 +9,325 @@ MetaDescription: Tutorial that walks you through creating a VS Code extension th
 
 # Tutorial: Generate AI-powered code annotations by using the Language Model API
 
-In this tutorial, you'll learn ...
+In this tutorial, You'll learn how to create a VS extension to build an AI-powered Code Tutor. You use the Language Model (LM) API to generate suggestions to improve your code and take advantage of the VS Code extension APIs to integrate it seamlessly in the editor as inline annotatations that the user can hover over for more information. After you complete this tutorial, you will know how to implement custom AI features in VS Code.
+
+![VS Code displaying custom annotations from GitHub Copilot as annotations](./images/lm-api/code-tutor-annotations-gif.gif)
 
 ## Prerequisites
 
-## Step 1: do something
+You'll need the following tools and accounts to complete this tutorial:
+
+- [Visual Studio Code](https://code.visualstudio.com/download)
+- [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat)
```

Comment:
I suggest to link to the github.copilot extension, not the chat one. Since the copilot one will just bring in the chat extension.